numpy/core/fromnumeric.py

```diff
@@ -797,9 +797,10 @@ def sort(a, axis=-1, kind='quicksort', order=None):
     =========== ======= ============= ============ =======
        kind      speed   worst case    work space  stable
     =========== ======= ============= ============ =======
-    'quicksort'    1     O(n^2)            0          no
-    'mergesort'    2     O(n*log(n))      ~n/2        yes
-    'heapsort'     3     O(n*log(n))       0          no
+    'quicksort'    1     O(n^2)            0         no
+    'mergesort'    2     O(n*log(n))      ~n/2       yes
+    'stable'       2     O(n*log(n))      ~n/2       yes
```

Comment:
If `stable` is not documented as an alias of `mergesort`, then it seems kinda strange to document as having exactly the same properties of mergesort without explaining this is because it _is_ mergesort

Reply:
I think @jaimefrio would have something to say on this. He suggested that we could leave this information undocumented and then have the liberty to map 'stable' to the best algorithm based on the type of values being sorted, for example: if we have a count sort, we could point 'stable' at count sort.

This would also allow us to have labels like 'fastest'.